build-bot/readme.md

```diff
@@ -0,0 +1,16 @@
+# MDOLab BuildBot Setup
+
+This directory contains the script that should almost entirely automate the setup of a BuildBot for running Github actions for MDOLab repos.
+
+## Installation steps
+
+1. Install a minimal version of the latest Ubuntu LTS version on the machine, use the following when setting up the user account:
+   1. **Your name:** `mdolabbuildbot-N` (replace "N" is the next available buildbot number)
+   2. **Your computer's name:** `mdolabbuildbot-N`
+   3. **Pick a username:** `mdolab_mnt`
+   4. **Choose a password:** `Civic-Hangover2-Virtual`
+2. Download this directory to somewhere like `/home/buildbot`
```

Comment:
This should probably be `/home/mdolab_mnt` to match the user

Reply:
Done